docs/essentials/widget_language.md

```diff
@@ -372,7 +371,7 @@ finally {
 }
 ```
 
-`let_watch!` is a more convenient macro if you want to subscribe an expression. Like `watch!`, it converts an expression to a value stream, but also auto unsubscribe this when the root of `widget!` over. We use `let_watch!` instead of `watch!`, `.unsubscribe_when_dropped()` and `move_to_widget!(guard)` not necessary anymore.
+`let_watch!` is a more convenient macro if you want to subscribe to an expression. Like `watch!`, it converts an expression to a value stream, but also auto unsubscribes when the **root of `widget!` over**(I don't understand this part) . When we use `let_watch!` instead of `watch!`, `.unsubscribe_when_dropped()` and `move_to_widget!(guard)` are not required anymore.
```

Comment:
My fault!  It should be:

`let_watch!` is a more convenient macro if you want to subscribe an expression. It converts an expression to a value stream and automatically release the subscription when the root of `widget!` is over its lifetime. The `let_watch!(...).subscribe(...);` generates: 
```
 let guard = watch!(...)
    .subscribe(move |_| { ... })
    .unsubscribe_when_dropped();
  move_to_widget!(guard);
```